packages/javascript-mutator/src/helpers/NodeGenerator.ts

```diff
@@ -1,16 +1,10 @@
 import * as types from '@babel/types';
 
 export class NodeGenerator {
-  public static createBooleanLiteralNode(originalNode: types.Node, value: boolean): types.BooleanLiteral {
+  public static createMutatedCloneWithProperties(originalNode: types.Node, props: any): types.Node {
```

Comment:
there has to be `any`?
If it's an object, at least consider ``{ [key: string]: any }``

Reply:
> there has to be `any`?

Just me still used to plain-old JavaScript

> If it's an object, at least consider `{ [key: string]: any }`

done

Unfortunately I cannot think of anything better right now, wouldn't mind any ideas here.